index.js

```diff
@@ -0,0 +1,113 @@
+import { darkTheme, lightTheme } from "./theme.js"
+import { createNewSideNote, generateId, shortenContent } from "./utils.js"
+
+let storedNotes = JSON.parse(localStorage.getItem("notes"))
```

Comment:
For a small app with one key. Storing the keys seems irrelevant. But for larger apps, it would be a good thing to track. :)

Thanks for pointing out the `null` case. Totally forgot to handle that case.